installer/manifests/keptn/keptn-ingress.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: ingress.placeholder
+  name: keptn-ingress
+  namespace: keptn
+spec:
+  tls:
+    - hosts:
+        - '*.keptn.domain.placeholder'
+      secretName: sslcerts
+  rules:
+    - host: '*.keptn.domain.placeholder'
```

Comment:
Careful, this change is potentially also affecting the expose Keptn's Bridge task. I wonder if we can still write `host: 'api.keptn.domain.placeholder'` here?

Reply:
Agree, let's use `api.keptn` as host.